autode/species/complex.py

```diff
@@ -227,23 +248,31 @@ def __init__(self, *args, name='complex'):
 
         Keyword Arguments:
             name (str):
+            do_init_translation (bool): Translate molecules initially such
+                                        that they do not overlap
         """
         self.molecules = args
         self.molecule_atom_indexes = []
 
         # Calculate the overall charge and spin multiplicity on the system and
         # initialise
         complex_charge = sum([mol.charge for mol in self.molecules])
-        complex_mult = sum([mol.mult for mol in self.molecules]) - (len(self.molecules) - 1)
+        complex_mult = (sum([mol.mult for mol in self.molecules])
+                        - (self.n_molecules - 1))
 
         complex_atoms = []
         for mol in self.molecules:
             complex_atoms += deepcopy(mol.atoms)
 
-        super().__init__(name=name, atoms=complex_atoms, charge=complex_charge, mult=complex_mult)
+        super().__init__(name=name,
+                         atoms=complex_atoms,
+                         charge=complex_charge,
+                         mult=complex_mult)
 
-        self.solvent = self.molecules[0].solvent if len(self.molecules) > 0 else None
+        if do_init_translation:
+            self._init_translation()
 
+        self.solvent = self.molecules[0].solvent if len(self.molecules) > 0 else None
```

Comment:
Should there be an assertion here that all the molecules have the same solvent? Or is it checked elsewhere/assumed?

Reply:
Yes there should – thanks